ElectronClient/app.js

```diff
@@ -322,6 +322,14 @@ class Application extends BaseApplication {
 		});
 	}
 
+	checkLayout(layout) {
+		if (layout === 'viewer') {
+			this.focusElement_('noteTextViewer');
+		} else {
+			this.focusElement_('noteBody');
+		}
+	}
```

Comment:
It doesn't seem like the right way to implement this, because you add a new element that doesn't really exist (noteTextViewer).

Instead you should modify the command handler `if (command.name === 'focusElement' && command.target === 'noteBody') {` in NoteText.jsx. Then focus either the editor or the viewer depending on what's currently visible.

Reply:
Alright, will do! Thanks.